cmd/der2ascii/writer_test.go

```diff
@@ -146,7 +158,12 @@ var derToASCIITests = []convertFuncTest{
 	// Empty BIT STRINGs do not emit extra whitspace.
 	{
 		[]byte{0x03, 0x01, 0x00},
-		"BIT_STRING { `00` }\n",
+		"BIT_STRING { b`` }\n",
+	},
+	// "Empty" BIT STRINGs that assert they have padding emitted as bytes.
```

Comment:
Perhaps just "Invalid BIT STRINGs are encoded as bytes" and also include one that claims 8 padding bits.